spec/expressions.md

```diff
@@ -804,19 +855,19 @@ A *lower-bound inference from* a type U *to* a type V is made as follows:
 
 -   V is an array type V~1~\[…\]and U is an array type U~1~\[…\]of the same rank
 
--   V is one of IEnumerable&lt;V~1~&gt;, ICollection&lt;V~1~&gt;, IReadOnlyList&lt;V~1~&gt;&gt;, IReadOnlyCollection&lt;V~1~&gt; or IList&lt;V~1~&gt; and U is a single-dimensional array type U~1~\[\]
+-   V is one of IEnumerable<V~1~>, ICollection<V~1~>, IReadOnlyList<V~1~>>, IReadOnlyCollection<V~1~> or IList<V~1~> and U is a single-dimensional array type U~1~\[\]
```

Comment:
again, this should be code fenced, not subscripts. Similar edits should be made to the remaining bullet items.

```suggestion
-   `V` is one of `IEnumerable<Vi>`, `ICollection<Vi>`, `IReadOnlyList<Vi>>`, `IReadOnlyCollection<Vi>` or `IList<Vi>` and `U` is a single-dimensional array type `Ui[]`
```